app/gui/src/presenter/project.rs

```diff
@@ -175,6 +178,17 @@ impl Model {
         })
     }
 
+    /// Notification from shortcut manager that the handled shortcut has changed.
+    /// It is either the name of the command that was triggered or `None` when handling of the
+    /// command was completed.
```

Comment:
A bit of nitpicking: I don't know why, but I miss a word here.
```suggestion
    /// last command was completed.
```